contracts/src/v0.8/dev/KeeperRegistryLogic.sol

```diff
@@ -8,15 +8,19 @@ import "./KeeperRegistryBase.sol";
  */
 contract KeeperRegistryLogic is KeeperRegistryBase {
   /**
+   * @param paymentModel one of Default, Arbitrum, Optimism
+   * @param registryGasOverhead the registry gas overhead
```

Comment:
nit. "the gas overhead used by registry in performUpkeep"

Reply:
Done